setup.py

```diff
@@ -576,31 +585,29 @@ def pxd(name):
                      '_libs/tslibs/conversion']},
     '_libs.tslibs.parsing': {
         'pyxfile': '_libs/tslibs/parsing',
-        'pxdfiles': ['_libs/src/util',
-                     '_libs/src/khash']},
+        'include': numpy_incls},
     '_libs.tslibs.strptime': {
         'pyxfile': '_libs/tslibs/strptime',
         'pxdfiles': ['_libs/src/util',
-                     '_libs/tslibs/nattype'],
-        'depends': tseries_depends,
```

Comment:
same

Reply:
same

Reply:
tseries_depends already incldes np_datetime, so no need to add this explicty.

Reply:
i think this one needs to change back

Reply:
From the diff it isn't clear which file you're referring to.  Best guess is tslibs.strptime?

Reply:
for tslibs.strptime, still needs to have tseries_depends, np? (e.g. np_datetime is there)